patches/minecraft/net/minecraft/world/entity/Entity.java.patch

```diff
@@ -519,6 +564,17 @@
        return this.f_19799_.getDouble(p_204037_);
     }
  
+@@ -2903,6 +_,10 @@
+       return new ClientboundAddEntityPacket(this);
+    }
+ 
++   /**
++    * @deprecated Can be overridden. Use {@link #getDimensionsForge(Pose)} instead.
++    */
++   @Deprecated
```

Comment:
Same comments as on getEyeHeight(Pose, Dims)

Reply:
Actually, we don't appear to have an accessor for this, so we may want to add one.  Otherwise the event-computed dimensions aren't really exposed.

Reply:
This method is public so I don't know what we could need an accessor for?